client/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx

```diff
@@ -47,6 +44,22 @@ const JetpackComMasterbar: React.FC< Props > = ( { pathname } ) => {
 	const jetpackSaleCoupon = useSelector( getJetpackSaleCoupon );
 	const isLoggedIn = useSelector( isUserLoggedIn );
 	const user = useSelector( getCurrentUser );
+
+	const shoppingCartTracker = useShoppingCartTracker();
+
+	const onRemoveProductFromCart = ( productSlug: string ) => {
+		shoppingCartTracker( 'calypso_jetpack_shopping_cart_remove_product', {
+			productSlug,
+		} );
+	};
+
+	const goToCheckout = ( siteSlug: string ) => {
+		shoppingCartTracker( 'calypso_jetpack_shopping_cart_checkout_from_cart', {
+			addProducts: true,
+		} );
+		page( `https://wordpress.com/checkout/${ siteSlug }` );
```

Comment:
When clicking the "Go to checkout" button, I'm seeing an exception thrown in the console:
<img width="1568" alt="Screenshot on 2022-12-16 at 14-40-11" src="https://user-images.githubusercontent.com/11078128/208176904-2e869ce2-a6c4-4c07-bd56-0354761e04d9.png">

I'm pretty sure it's because this call to `page( path );`
`page()` uses `history.pushState()` which will throw an exception if the new url is not of the same origin as the old url (documented [here](https://developer.mozilla.org/en-US/docs/Web/API/History/pushState#url). 

Instead, since we need to route to a url of different origin, I believe we're forced to use `window.location.href` instead.

Reply:
Yes, thanks for catching this. Updated.